apps/token-manager/test/tokenmanager.js

```diff
@@ -291,13 +293,13 @@ contract('Token Manager', ([root, holder, holder2, anyone]) => {
 
             it("cannot call proxyPayment() from outside of the token's context", async () => {
                 const value = 10
-                const prevTokenManagerBalance = (await getBalance(tokenManager.address)).toNumber()
+                const prevTokenManagerBalance = parseInt(await getBalance(tokenManager.address), 10)
```

Comment:
We should prefer serializing and comparing as strings, as otherwise a large value here will be imprecise and result in flaky tests.